SMLHelper/SMLHelper.csproj

```diff
@@ -20,113 +21,99 @@
   <PropertyGroup>
     <StartupObject />
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'SUBNAUTICA|AnyCPU'">
-    <OutputPath>bin\SUBNAUTICA\</OutputPath>
-    <DefineConstants>TRACE;DEBUG;SUBNAUTICA</DefineConstants>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'SN.STABLE|AnyCPU'">
```

Comment:
@MrPurple6411 @AlexejheroYTB So, why you don't want periods in names? I'm using them quite often, both in folders and configuration names, didn't have any problems with them. There is also SMLHelper.Tests folder right in the root.